metashare/api/views.py

```diff
@@ -198,13 +193,14 @@ def list(self, request, *args, **kwargs):
         # XXX: I am apprehensive about the possibility of flooding the
         # worker queues easily this way:
         filters = {
-            "owner": request.user,
             "org_type": SCRATCH_ORG_TYPES.Dev,
             "url__isnull": False,
             "delete_queued_at__isnull": True,
             "currently_capturing_changes": False,
             "currently_refreshing_changes": False,
         }
+        if not request.GET.get("get_unsaved_changes"):
```

Comment:
```suggestion
        if not request.query_params.get("get_unsaved_changes"):
```
See https://www.django-rest-framework.org/api-guide/requests/#query_params